worlds/pokemon_crystal/data/regions.json

```diff
@@ -0,0 +1,3337 @@
+{
```

Comment:
It does add 1500 newlines, but auto-formatting this file makes it so much more readable because of the consistency imo.

Reply:
Which autoformatter? I have been fighting against PyCharm's ugly JSON preferences this whole project

Reply:
An unmodified install of VSCode has Right Click > Format Document for JSON. I'm not sure where it derives its rules. I wasn't aware Pycharm had opinions on JSON formatting.